features/steps/steps.py

```diff
@@ -221,3 +234,46 @@ def step_impl(context, related, relating, other_entity, condition):
                         errors.append(instance_structure_error(inst, rel.RelatingObject))
 
     handle_errors(context, errors)
+
+
+@then('Each {entity} {condition} be {directness} contained in {other_entity}')
+def step_impl(context, entity, condition, directness, other_entity):
+    stmt_to_op = ['must', 'must not']
+    assert condition in stmt_to_op
+
+    stmt_about_directness = ['directly', 'indirectly', 'directly or indirectly', 'indirectly or directly']
+    assert directness in stmt_about_directness
+
+    errors = []
+
+    if context.instances and getattr(context, 'applicable', True):
+        for ent in context.model.by_type(entity):
+            if len(ent.ContainedInStructure) > 0:
+                containing_relation = ent.ContainedInStructure[0]
+                relating_spatial_element = containing_relation.RelatingStructure
+                is_directly_contained = getattr(relating_spatial_element, 'is_a')(other_entity)
```

Comment:
`getattr(relating_spatial_element, 'is_a')`

is equivalent to

`relating_spatial_element.is_a`

I'd say only use `getattr(object, name[, default])` if:
- name is a variable
- default is used so that you don't get a KeyError on non-existant attr